assets/pages/contributions-landing/actions/contributionsLandingActions.js

```diff
@@ -10,7 +10,7 @@ import type { Contrib, Amount } from 'helpers/contributions';
 export type Action =
   | { type: 'CHANGE_CONTRIB_TYPE', contribType: Contrib }
   | { type: 'CHANGE_CONTRIB_AMOUNT', amount: Amount }
-  | { type: 'CHANGE_CONTRIB_AMOUNT_RECURRING', amount: Amount }
+  | { type: 'CHANGE_CONTRIB_AMOUNT_MONTHLY', amount: Amount }
```

Comment:
Is there a reason for not having an _ANNUAL entry in this list?

Reply:
Seems to work without it, but I'll investigate. Good spot!